pkgs/applications/blockchains/dogecoin/default.nix

```diff
@@ -1,12 +1,14 @@
 { lib, stdenv , fetchFromGitHub
 , pkg-config, autoreconfHook
 , db5, openssl, boost, zlib, miniupnpc, libevent
-, protobuf, util-linux, qt4, qrencode
-, withGui }:
+, protobuf, qtbase ? null
+, wrapQtAppsHook ? null, qttools, qmake ? null, qrencode
```

Comment:
I believe qttools is only available as a part of `libsForQt5`.

Reply:
This is why `libsForQt5.callPackage` is used

Reply:
Ah, I didn't check that, my bad. Dunno why it's not being found then. I'll have to take another look later.